Source/Scene/FXAA.js

```diff
@@ -119,7 +140,15 @@ define([
         }
 
         if (!defined(this._command)) {
-            this._command = context.createViewportQuadCommand(FXAAFS, {
+            var fs =
+                '#define FXAA_PC 1\n' +
+                '#define FXAA_WEBGL_1 1\n' +
```

Comment:
Are there WebGL 2 optimizations that we should roadmap in #797?  If so, please update that issue.